### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Best to review each commit separately

### CODE REVIEW

File: src/show.cpp [hunk 1]
```diff
@@ -59,7 +59,11 @@ int main(int argc, char **argv) {
         }
 
         cv::imshow("visualization", depth_image);
-        cv::waitKey(10);
+        char key = cv::waitKey(10);
+        if (key == 'q')
+        {
+            break;
```
Comment (ar13pit) [inline src/show.cpp:65]: Why not call `cv::DestroyAllWindows()` here as well?

File: src/sensors/DepthCamera.cpp [hunk 2]
```diff
@@ -76,7 +76,7 @@ void DepthCamera::render(const RenderOptions& opt, RenderResult& res) const {
     }
 
     int i_triangle = 0;
-    for(std::vector<TriangleI>::const_iterator it_tri = triangles.begin(); it_tri != triangles.end(); ++it_tri) {
+    for(std::vector<TriangleI>::const_iterator it_tri = triangles.cbegin(); it_tri != triangles.cend(); ++it_tri) {
```
Comment (ar13pit) [inline src/sensors/DepthCamera.cpp:79]: Also change the iterator type to auto

**Review by ar13pit — CHANGES_REQUESTED** [review]

File: src/sensors/DepthCamera.cpp (hunk 2 above)
Comment (MatthijsBurgh) [reply src/sensors/DepthCamera.cpp:79]: I would like to keep it this way.

File: src/show.cpp (hunk 1 above)
Comment (MatthijsBurgh) [reply src/show.cpp:65]: It will close automatically

**Review by ar13pit — APPROVED** [review]

**Pull request merged**